app/views/layouts/application.html.erb

```diff
@@ -156,10 +156,11 @@
                 ) %>
               </p>
               <p>
-                RubyGems.org is made possible by
-                <%= link_to "many sponsors", page_path("sponsors") %>,
-                with development funded by the Ruby trade association,
-                <%= link_to "Ruby Together", "https://rubytogether.org/?source=rubygems" %>.
+                 RubyGems.org is made possible through a partnership with the greater Ruby community.
+                <%= link_to "Fastly" "https://www.fastly.com/") %> provides bandwidth and CDN support,
+                <%= link_to "Ruby Central" "http://www.rubycentral.org/") %> covers infrastructure costs, and
+                <%= link_to "Ruby Together" "https://rubytogether.org/?source=rubygems" %> funds ongoing development and ops work.
+                <%= link_to "Learn more about our sponsors and how they work together." page_path("sponsors") %>
```

Comment:
Hi, syntax here is all over the place. can we please fix it?
[link_to](http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to)